src/txkube/testing/strategies.py

```diff
@@ -252,6 +252,16 @@ def podspecs():
     """
     return builds(
         v1.PodSpec,
+        activeDeadlineSeconds=one_of(
+            none(),
+            integers(min_value=0, max_value=2 ** 63 - 1),
```

Comment:
Yep, it makes no sense, but it's how Kubernetes works.  This value is a 64 bit signed integer the minimum acceptable value of which is 0...  I'll add a comment and maybe a link to the spec.